src/v/cluster/data_migration_backend.cc

```diff
@@ -193,6 +193,7 @@ ss::future<> backend::loop_once() {
 }
 
 ss::future<> backend::work_once() {
+    vlog(dm_log.info, "begin backend work cycle");
```

Comment:
@bashtanov perhaps this is too verbose should be debug level?

Reply:
ok will change in one of subsequent PRs